src/types/recommendations.d.ts

```diff
@@ -62,3 +66,22 @@ export interface RecommendationsResultType extends Record<string, any> {
     [key: string]: any;
   };
 }
+
+export interface RecommendationPodsResponse extends Record<string, any> {
+  pods: RecommendationPod[];
+  total_count: number;
+}
+
+export interface PodStrategy extends Record<string, any> {
+  id: string;
+  display_name: string;
+}
+export interface RecommendationPod extends Record<string, any> {
+  strategy: PodStrategy;
+  id: string;
+  display_name: string;
+  name: string;
+  created_at: string;
+  updated_at: string;
+  metadata_json: Record<string, any>;
```

Comment:
I'm not the Typescript expert but Record<> means object, right? In the response it looks like an array 🤔 
`metadata_json: []`

Reply:
Great catch the example is wrong here. fixed